File: linear_search/LinearSearch.java
import java.util.*;

/**
 * LinearSearch
 */
public class LinearSearch {

  public static void main(String[] args) {
    Integer inputArray[] = {13, 5, 17, 2, 11, 35};
    Integer targetNum = 11;
    boolean targetFoundFlg = false;

    long start = System.nanoTime();

    for(Integer inputNum : inputArray) {
      if(targetNum == inputNum) {
        targetFoundFlg = true;
        System.out.println("The target number(" + targetNum + ") was found.");
        break;
      }
    }

    if(targetFoundFlg != true) {
      System.out.println("The target number (" + targetNum + ") was not found...");
    }

    long end = System.nanoTime();

    System.out.println("Exec time: " + (end - start) / 1000000f + " ms");
  }
}

File: linear_search/LinearSearch.rb
require 'time'

class LinearSearch
  input_arr = [13, 5, 17, 2, 11, 35]
  target_num = 11
  target_find_flg = false

  start_time = Time.now

  input_arr.each do |input_num|
    if target_num == input_num
      target_find_flg = true
      puts "The target number(#{target_num}) was found."
      break;
    end
  end

  if target_find_flg == false
    puts "The target number(#{target_num}) was not found..."
  end

  end_time = Time.now

  puts "Exec time: #{(end_time - start_time) * 100000} ms"
end